engine/classes/Elgg/TimeUsing.php

```diff
@@ -0,0 +1,47 @@
+<?php
+namespace Elgg;
+
+use DateTime;
+
+/**
+ * Adds methods for fixing the current time (for testing)
```

Comment:
In the future we may want to clarify that we are locking time and not "fixing" it. I was confused why time needs fixing